src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +50,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
+  }
+
+  /**
+   * Loads a map object that currently displays a hardcoded location.
+   * @param {Object} coordinates - the LatLng object of user's current location
+   * ToDo(kristenmason@): implement grant location access and modify
+   * function to pass in current position (currently coords default to GooglePlex)
+   * @private 
+  */
+  loadMap_(coordinates = new google.maps.LatLng(37.422004,-122.081402)) { 
+    const map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
Will these variables eventually be used, or is the constructor the only important part?

If you don't actually need these variables, don't bother creating them just call the constructor

Reply:
The current plan is to either pass the map object into at least one other function that's still in the works or declare it as a global variable so that all of my functions can access it-- if it ends up being that latter I'll move it!